src/libraries/CalldataDecoder.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {Call} from "./CallLib.sol";
+
+library CalldataDecoder {
+    using CalldataDecoder for bytes;
+    /// @notice equivalent to SliceOutOfBounds.selector, stored in least-significant bits
+
+    error SliceOutOfBounds();
```

Comment:
not used?

Reply:
ah yeah this was a pattern we kept to just show the custom error selector `SLICE_ERROR_SELECTOR` but I can just comment it for now